manifests/m/Microsoft/AzureDataCLI/20.3.5/Microsoft.AzureDataCLI.locale.en-US.yaml

```
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.AzureDataCLI
+PackageVersion: 20.3.5
+PackageLocale: en-US
+Publisher: Microsoft

```

```suggestion
Publisher: Microsoft Corporation
```
Needs to match what control panel sees.

---

Thanks for the suggestions and reasons behind them! @OfficialEsco